vivarium/core/process.py

```diff
@@ -120,6 +122,14 @@ class can provide a ``defaults`` class variable to specify the
       """
 
     defaults: Dict[str, Any] = {}
+    METHOD_COMMANDS = (
+        'initial_state', 'generate_processes', 'generate_steps',
+        'generate_topology', 'generate_flow', 'merge_overrides',
+        'calculate_timestep', 'is_step', 'get_private_state',
+        'ports_schema', 'next_update', 'update_condition')
```

Comment:
Quality list. I don't see anything missing (!) didn't realize the `Process` API had grown so much.